src/gui/SampleLoader.cpp

```diff
@@ -68,14 +68,14 @@ QString SampleLoader::openAudioFile(const QString& previousFile)
 		const auto name = QString::fromStdString(audioType.name);
 		const auto extension = QString::fromStdString(audioType.extension);
 
-		fileTypes.append(QString{"%1-Files (*.%2)"}.arg(name, extension));
+		fileTypes.append(QObject::tr(qPrintable(QString{"%1-Files (*.%2)"}.arg(name, extension))));
 
-		allFileTypes.append(QString{"*.%2"}.arg(extension));
+		allFileTypes.append(QObject::tr(qPrintable(QString{"*.%2"}.arg(extension))));
```

Comment:
Is the use of `qPrintable` correct here? I'd imagine that translated strings need to be known at compile-time.

- "-Files" could be translated (and probably reworded as " files" without the dash)
- On line 62, "All Audio-Files (" is not translated, and could probably be reworded as "All audio files ("

Reply:
> Is the use of qPrintable correct here? I'd imagine that translated strings need to be known at compile-time.

I was also wondering this. I don't think this is right because of the temporary `QString`. It worked when I tested it and didn't segfault, but I don't think that's overall a safe thing to do. It could break on other systems. Translating only the "Files" and "All audio files" parts seems reasonable.